src/mcts/search.cc

```diff
@@ -65,9 +67,14 @@ void Search::PopulateUciParams(OptionsParser* options) {
   options->Add<IntOption>(kMiniBatchSizeStr, 1, 1024, "minibatch-size") = 1;
   options->Add<IntOption>(kMaxPrefetchBatchStr, 0, 1024, "max-prefetch") = 32;
   options->Add<FloatOption>(kCpuctStr, 0.0f, 100.0f, "cpuct") = 1.2f;
-  options->Add<FloatOption>(kTemperatureStr, 0.0f, 100.0f, "temperature") =
-      0.0f;
-  options->Add<IntOption>(kTempDecayMovesStr, 0, 100, "tempdecay-moves") = 0;
+  options->Add<FloatOption>(kMoveTempStr, 0.0f, 100.0f, "movetemperature") = 0.0f;
+  options->Add<FloatOption>(kMoveTempDecayStr, 1.0f,
+                            std::numeric_limits<float>::max(),
```

Comment:
It's better to set limits to something a little bit above reasonable rather than to totally unreasonable.
That way it's easier to see in --help and in uci UI (although for float it's not the case) what is the sensible range.
Also, it looks nicer in --help text.

Reply:
Yea that's one thing I knew wasn't optimal. The problem is, what's a suitably large max? Being logarithmic, values like 1e6 or 1e8 are perfectly reasonable. 1e10? 1e15?